Demo.md

```diff
@@ -29,7 +35,14 @@ Download your configuration YAML file from the Contrast environment by selecting
 
 ## Step 4 - Start the DEMO environment
 
-`./demo.sh`
+`./demo.sh $FRAMEWORK`
```

Comment:
I would add an example here since the sales team is the primary user of the demo environment and it's better to spell out anything like this. Something like "(example: `./demo.sh std`)"